### DISCUSSION TITLE
See the leaderboard with the console

### DISCUSSION THREAD

**Discussion by ax-now** [ReactiveDrop/reactivedrop_public_src#322] [Ideas]
Yo, it would be cool to have a console command to see the current leaderboard in-game by console.
something like: rd_show_leaderboard

This would allow you to see the leaderboard directly in game without going to the rd site and looking for the affiliate challenge and also to see the leaderboard while the player(s) are playing. :P

This would display as the top 20 or 30 in the leaderboard so it wouldn't take too long in the console.

**Comment by BenLubar** [comment]
Do you mean something like rd_lobby_debug_* but for leaderboards?
